README.md

```diff
@@ -75,15 +97,15 @@ token$credentials$refresh_token
 
 ### Setting up Calendly
 
-1. To set up calendly authorization [go here](https://calendly.com/integrations/api_webhooks) and click "Generate Token". 
+1. To set up calendly authorization [go here](https://calendly.com/integrations/api_webhooks) and click "Generate Token".
 2. Underneath "Choose a name for this token" pick a name that will remind you of this project.
 3. Click "Create Token" and it will send you an authorization code to your email.
 4. Click "Copy Token" and keep this handy.
 5. Return to your metricminer dashboard repository and go to `Settings` > `Secrets and variables` > `Actions`.
 6. Click on `New repository secret`. Name your new secret *exactly* `METRICMINER_CALENDLY`
 7. Paste your token into the `Secret` box and then click the green "Add secret" button.
 
-After you've set up authorization you'll need to check the following items in the `_config_automation.yml` file. 
+After you've set up authorization you'll need to check the following items in the `_config_automation.yml` file.
 
 - [ ] Make sure that `refresh-calendly` is set to "yes".
```

Comment:
should it be specified that this is within the config file? Same comment for each section throughout that uses this type of instruction

Reply:
Is the sentence that precedes it not clear? I suppose we can add it to each checklist item? Is that what you mean?
```suggestion
- [ ] In the `config_automation.yml` fileMake sure that `refresh-calendly` is set to "yes".
```

Reply:
Done!